pandas/tseries/tests/test_resample.py

```diff
@@ -16,24 +16,25 @@
 from pandas.core.common import ABCSeries, ABCDataFrame
 from pandas.core.groupby import DataError
 from pandas.tseries.frequencies import MONTHS, DAYS
+from pandas.tseries.frequencies import to_offset
 from pandas.tseries.index import date_range
 from pandas.tseries.offsets import Minute, BDay
 from pandas.tseries.period import period_range, PeriodIndex, Period
 from pandas.tseries.resample import (DatetimeIndex, TimeGrouper,
                                      DatetimeIndexResampler)
-from pandas.tseries.frequencies import to_offset
 from pandas.tseries.tdi import timedelta_range
 from pandas.util.testing import (assert_series_equal, assert_almost_equal,
-                                 assert_frame_equal)
+                                 assert_frame_equal, assert_index_equal)
+from pandas._period import IncompatibleFrequency
 
 bday = BDay()
 
 # The various methods we support
-downsample_methods = ['min', 'max', 'first', 'last', 'sum', 'mean', 'sem',
-                      'median', 'prod', 'var', 'ohlc']
-upsample_methods = ['count', 'size']
-series_methods = ['nunique']
-resample_methods = downsample_methods + upsample_methods + series_methods
+downsample_methods = {'min', 'max', 'first', 'last', 'sum', 'mean', 'sem',
```

Comment:
why did you change these? I don't think ordering is preserved. Leave them as lists.